### PR TITLE
Seperate backends

### PR DESCRIPTION
**Pull request by henkieee**
The branch with the code to pick the right backend

### CODE REVIEW

File: app/scripts/locals.js [hunk 1]
```diff
@@ -614,6 +614,9 @@ define(
           states: 'States',
           saveTeam: 'Save team',
           save: 'Save',
+          systemUnavailable: "The system is temporarily unavailable, try again in a few seconds",
+          noBackend: "The defined backend cannot be found",
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Should be 'back-end'.

File: app/scripts/locals.js [hunk 2]
```diff
@@ -614,6 +614,9 @@ define(
...
+          statusCodeNotRegonized: "Statuscode not regonized",
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Should be 'status code'.

File: app/scripts/locals.js [hunk 3]
```diff
@@ -2717,6 +2725,11 @@ define(
           states: 'Status',
           saveTeam: 'Team opslaan',
           save: 'Opslaan',
+          //start TODO
+          systemUnavailable: "Het systeem is tijdelijk buiten gebruik, probeer het naar een paar seconde nog een keer.",
```
Comment (leonjoosse) [inline app/scripts/locals.js]: 'seconde' should be 'seconden', as it is a plural.
I would actually make this sentence more like 'Het systeem is tijdelijk niet bereikbaar, probeer opnieuw in te loggen. Als het probleem blijft voorkomen, neem dan contact op met uw systeembeheerder.', as your sentence sounds too informal, in my opinion.

File: app/scripts/services/settings.js [hunk 4]
```diff
@@ -1,168 +1,189 @@
...
+        function success(results)
+        {
+          var result = self.getResultOnStatusCode(results);
+          self.setBackEnd(backEnds[result.index]);
+          deferred.resolve(result);
+        });
+        return deferred.promise;
+      },
+      /**
+       * Depending on the order of status codes,
```
Comment (leonjoosse) [inline app/scripts/services/settings.js]: Would be nice to describe what this order is about and what each entry in this order means. It is really vague.

File: app/scripts/services/settings.js [hunk 5]
```diff
@@ -1,168 +1,189 @@
...
+       * the first result in that order will return validated
+       * @param results All the requests with the different backends
```
Comment (leonjoosse) [inline app/scripts/services/settings.js]: I would write something like 'An array of objects, where each entry represents the http status code of a /login request.'

**Comment by henkieee** [comment]
I've added some documentation and made some changes in the spelling

**Pull request merged**